packages/web/src/features/rental-business/frames/RentalInfoFrame/RentalInfoSecondFrame.tsx

```diff
@@ -21,53 +29,24 @@ const StyledCardInner = styled.div`
   align-self: stretch;
 `;
 
-const Easel: React.FC<RentalFrameProps> = () => (
-  <Typography type="p">대여 개수</Typography>
-);
-
-const Vacuum: React.FC<RentalFrameProps> = ({ rental, setRental }) => (
-  <>
-    <Typography type="p">청소기 종류</Typography>
-    <Radio
-      value={rental?.vacuum || "cordless"}
-      onChange={value =>
-        setRental({
-          ...rental,
-          vacuum: value,
-        })
-      }
-    >
-      <Radio.Option value="cordless">무선 청소기</Radio.Option>
-      <Radio.Option value="corded">유선 청소기</Radio.Option>
-    </Radio>
-  </>
-);
-
-const HandCart: React.FC<RentalFrameProps> = () => (
-  <>
-    <Typography type="p">롤테이너 개수</Typography>
-    <Typography type="p">대형 개수</Typography>
-    <Typography type="p">중형 개수</Typography>
-    <Typography type="p">소형 개수</Typography>
-  </>
-);
-
-const Mat: React.FC<RentalFrameProps> = () => (
-  <Typography type="p">대여 개수</Typography>
-);
-
-const Tool: React.FC<RentalFrameProps> = () => (
-  <>
-    <Typography type="p">전동 드릴 세트 개수</Typography>
-    <Typography type="p">드라이버 개수</Typography>
-    <Typography type="p">슈퍼글루 개수</Typography>
-    <Typography type="p">니퍼 개수</Typography>
-    <Typography type="p">플라이어 개수</Typography>
-    <Typography type="p">롱노즈플라이어 개수</Typography>
-  </>
-);
+const ResetTitleWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+  gap: 20px;
+`;
+
+const FlexGrowTypography = styled.div`
+  flex-grow: 1;
+`;
+
+const NoneRental: React.FC<RentalFrameProps> = () => <>none</>;
 
 const rentals = {
+  none: {
+    info: "대충 대여 기간 먼저 선택해야 한다는 안내문구 어딘가에",
+    component: NoneRental,
+  },
```

Comment:
요 정도 안밖에 떠오르지 않는군 그치만 best는 아닌 것 같아
```suggestion
  none: {
    info: "대충 대여 기간 먼저 선택해야 한다는 안내문구 어딘가에",
    // eslint-disable-next-line react/jsx-no-useless-fragment
    component: <></>,
  },
```

Reply:
옹 저거 나도 똑같은 고민을 하다가 eslint disable 웬만하면 안 달고 싶어서 저렇게 뒀어
먼가 안 예쁘면 info/component를 아예 분리해버릴 수도 있을 것 같은데 마지막 confirm 프레임 하면서 더 고민해볼게